packages/components/psammead-media-player/src/index.jsx

```diff
@@ -91,6 +91,7 @@ CanonicalMediaPlayer.propTypes = {
     duration: string,
     durationSpoken: string,
     type: oneOf(['video', 'audio']),
+    guidanceMessage: string,
```

Comment:
We need documentation of this new prop in the [README](https://github.com/bbc/psammead/blob/latest/packages/components/psammead-media-player/README.md)

Reply:
I've noticed that other props haven't been documented so I've created a separate issue for those https://github.com/bbc/psammead/issues/2541.

Reply:
There was an issue to do that here https://github.com/bbc/psammead/issues/2475 - feel free to merge/delete one. cc @jamesdonoh

Reply:
> There was an issue to do that here #2475 - feel free to merge/delete one.
> Cc @jamesdonoh

I've merged them into yours